_ml-commons-plugin/api.md

```diff
@@ -0,0 +1,968 @@
+            "entities" : [
+              {
+                "key" : [
+                  "attr0"
+                ],
+                "contribution_value" : 3.0,
+                "base_value" : 2.0,
+                "new_value" : 5.0
+              },
+              {
+                "key" : [
+                  "attr1"
+                ],
+                "contribution_value" : 3.0,
+                "base_value" : 3.0,
+                "new_value" : 6.0
+              }
+            ]
+          },
+          {
+            "start_time" : 1620975600000,
+            "end_time" : 1621062000000,
+            "overall_aggregate_value" : 105.0,
+            "entities" : [
+              {
+                "key" : [
+                  "attr0"
+                ],
+                "contribution_value" : 4.0,
+                "base_value" : 2.0,
+                "new_value" : 6.0
+              },
+              {
+                "key" : [
+                  "attr1"
+                ],
+                "contribution_value" : 4.0,
+                "base_value" : 3.0,
+                "new_value" : 7.0
+              }
+            ]
+          },
+          {
+            "start_time" : 1621062000000,
+            "end_time" : 1621148400000,
+            "overall_aggregate_value" : 115.0,
+            "entities" : [
+              {
+                "key" : [
+                  "attr0"
+                ],
+                "contribution_value" : 5.0,
+                "base_value" : 2.0,
+                "new_value" : 7.0
+              },
+              {
+                "key" : [
+                  "attr1"
+                ],
+                "contribution_value" : 5.0,
+                "base_value" : 3.0,
+                "new_value" : 8.0
+              }
+            ]
+          },
+          {
+            "start_time" : 1621148400000,
+            "end_time" : 1621234800000,
+            "overall_aggregate_value" : 125.0,
+            "entities" : [
+              {
+                "key" : [
+                  "attr0"
+                ],
+                "contribution_value" : 6.0,
+                "base_value" : 2.0,
+                "new_value" : 8.0
+              },
+              {
+                "key" : [
+                  "attr1"
+                ],
+                "contribution_value" : 6.0,
+                "base_value" : 3.0,
+                "new_value" : 9.0
+              }
+            ]
+          }
+        ]
+      }
+    }
+  ]
+}
+```
+
+## Search model
+
+Use this command to search models you're already created.
+
+
+```json
+POST /_plugins/_ml/models/_search
+{query}
+```
+
+
+### Example 1: Query all models
+
+```json
+POST /_plugins/_ml/models/_search
+{
+  "query": {
+    "match_all": {}
+  },
+  "size": 1000
+}
+```
+
+### Example 2: Query models with algorithm "BATCh_RCF"
+
+```json
+POST /_plugins/_ml/models/_search
+{
+  "query": {
+    "term": {
+      "algorithm": {
+        "value": "BATCH_RCF"
```

Comment:
The response bellow shows models with algorithm `FIT_RCF`, how about change this line to 
```
"value": "FIT_RCF"
```